sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/utils/crypto/CryptoNoOpHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.mobile.utils.crypto;
+
+import android.content.Context;
+
+import java.security.KeyStore;
+
+/**
+ * Handler that does not actually encrypt anything.
+ */
+class CryptoNoOpHandler implements CryptoHandler {
```

Comment:
That's because Java can specify only one class for inheritance. I feel `CryptoNoOpHandler` is like a default behavior. Conceptually it would override behaviors of super class in child classes.
Don't care to leave this as I mentioned it is my opinion.